doc/api/console.markdown

```diff
@@ -9,9 +9,9 @@ The module exports two specific components:
 
 * A `Console` class with methods such as `console.log()`, `console.error()` and
   `console.warn()` that can be used to write to any Node.js stream.
-* A global `console` instance configured to write to `stdout` and `stderr`.
-  Because this object is global, it can be used without calling
-  `require('console')`.
+* A global `console` instance configured to write to [`process.stdout`][] and
+  [`process.stderr`][]. Because this object is global, it can be used without
+  calling `require('console')`.
```

Comment:
@mscdex your thoughts? I agree with both of you, to say, so not sure ) Please decide )